packages/node/src/ethereum/api.ethereum.ts

```diff
@@ -90,6 +91,7 @@ export class EthereumApi implements ApiWrapper<EthereumBlockWrapper> {
 
   // Ethereum POS
   private supportsFinalization = true;
+  private blockConfirmations = yargsOptions.argv['block-confirmations'];
```

Comment:
It seems we cant easily extend NodeConfig from node core. Might be something worth doing later on